File: Data Structures & Algorithms/counting-bits/submission-0.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        res = [0] * (n+1)
        for i in range(n+1):
            cur,count = i,0
            while cur > 0:
                if cur % 2 == 1:
                    count += 1
                cur //= 2
            res[i] = count

        return res

File: Data Structures & Algorithms/majority-element-ii/submission-0.cpp
class Solution {
public:
    vector<int> majorityElement(vector<int>& nums) {
        int threshold = nums.size() / 3;
        unordered_map<int, int> count;
        for (int n : nums){
            count[n]++;
        }

        vector<int> res;
        for (auto& [key,count] : count){ // for k,v in map.items()
            if (count > threshold){
                res.push_back(key); //like python list.append()
            }
        }

        return res;
    }
};